### PR TITLE
Tweak to our bug issue template

### PR DESCRIPTION
**Pull request by abidlabs**
Small tweak to our bug issue template which adds the code syntax to the "Reproduction" section. I've noticed a lot of people dump code without the formatting, which makes it hard to parse. Hopefully, this will help!

image

Here's how it looks: https://github.com/abidlabs/gradio/issues/new?assignees=&labels=bug&projects=&template=bug_report_template.yml

### CODE REVIEW

**Review by freddyaboulton — APPROVED** [review]
Thanks @abidlabs !

**Comment by abidlabs** [comment]
added some quick timeouts to fix the flaky backend test

**Comment by abidlabs** [comment]
Will merge this in, thanks @freddyaboulton
